manifests/add_client.pp

```diff
@@ -21,45 +21,65 @@
 #   OPTIONAL - Max percent of RAM allowed for gpfs pagepool.
 #   Must be an integer between 10 and 90.
 #
-# @param nodeclasses
-#   OPTIONAL - list of nodeclasses to which this node should be added master_server
+# @param script_tgt_fn
+#   OPTIONAL - where to store the "add_client" bash script default set in module hiera
+#
+# @param ssh_private_key_contents
+#   private ssh key contents to enable ssh into master_server
 #
 # @param ssh_private_key_path
 #   OPTIONAL - path to store the gpfs private key (default set in module hiera)
 #
-# @param script_tgt_fn
-#   OPTIONAL - where to store the "add_client" bash script default set in module hiera
+# @param ssh_public_key_contents
+#   public ssh key contents to enable ssh into master_server
 #
-# @param mmsdrfs
-#   OPTIONAL - path to mmsdrfs command default set in module hiera
+# @param ssh_public_key_type
```

Comment:
This is the only new parameter.

Rest of parameter changes are just sorting the parameters to be alphabetical order.